libs/blockscout-service-launcher/src/test_database.rs

```diff
@@ -142,6 +142,27 @@ impl TestDbGuard {
             &name[..MAX_DATABASE_NAME_LEN - HASH_SUFFIX_STRING_LEN - 1]
         )
     }
+
+    pub async fn close_all(&self) -> (Result<(), DbErr>, Result<(), DbErr>) {
+        // remove `get_postgres_connection_pool` as soon as
+        // https://github.com/SeaQL/sea-orm/pull/2511 is available
+        // to work with other connections
+        self.conn_with_db
+            .get_postgres_connection_pool()
+            .close()
+            .await;
+        self.conn_without_db
+            .get_postgres_connection_pool()
+            .close()
+            .await;
```

Comment:
never use result of close() ? looks like function always returns (Ok, Ok)

Reply:
it was a temporary solution; now that my PR got merged it's the way it was intended